src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessClientExtensions.cs

```diff
@@ -55,6 +55,27 @@ public static class ExceptionlessClientExtensions {
             }
         }
 
+        internal static LogLevel GetLevel(this LogEventLevel log)
```

Comment:
We might want to ditch the extension method above `internal static LogLevel GetLevel(this LogEvent log) {` and just use this one.

Reply:
agree